xbmc/SortFileItem.h

```diff
@@ -40,6 +40,8 @@ typedef enum {
   SORT_METHOD_YEAR,
   SORT_METHOD_VIDEO_RATING,
   SORT_METHOD_DATEADDED,
+  SORT_METHOD_ALBUM_DATEADDED,
+  SORT_METHOD_ARTIST_DATEADDED,
```

Comment:
right, if that already working (never really looked into it) then it's OK for me.
Let's update the pydocs properly with a sorted list